scripts/actions/mobskills/hydro_wave.lua

```diff
@@ -0,0 +1,36 @@
+-----------------------------------
+--  Hydro Wave
+--  Description: Deals water damage to enemies around the caster.
+--  Type: Magical (Water)
+-----------------------------------
+local mobskillObject = {}
+
+mobskillObject.onMobSkillCheck = function(target, mob, skill)
+    return 0
+end
+
+mobskillObject.onMobWeaponSkill = function(target, mob, skill)
+    local duration = 30
+    local resist = applyResistanceAddEffect(mob, target, xi.element.WATER, 0)
```

Comment:
This always seems to return 1, resulting in a max duration every mobskill. Am I doing something wrong here?